aries-site/src/data/structures/components.js

```diff
@@ -128,6 +128,16 @@ export const components = [
     },
     relatedContent: ['Forms', 'TextArea', 'MaskedInput'],
   },
+  {
+    name: 'ToolTip',
```

Comment:
```suggestion
    name: 'Tooltip',
```
In Figma, we're not capitalizing the second t, so let's match that

Reply:
And then just scan other areas of page where we're saying Tooltip, and make sure it's "Tooltip" and not "ToolTip"